apps/pie-docs/src/__tests__/_11ty/shortcodes/__snapshots__/card.test.js.snap

```diff
@@ -22,43 +22,43 @@ exports[`card.js should return the expected HTML 2`] = `
+                <svg xmlns=\\"http://www.w3.org/2000/svg\\" role=\\"presentation\\" focusable=\\"false\\" fill=\\"#000\\" viewBox=\\"0 0 16 16\\" height=\\"21\\" width=\\"21\\" aria-hidden=\\"true\\" class=\\"c-pieIcon c-pieIcon--linkExternal\\"><path d=\\"M13.25 3.564 8.21 8.63l-.928-.928 5.093-5.066H9.662V1.324h3.833a1.093 1.093 0 0 1 1.067 1.093V6.25H13.25V3.564Z\\"></path><path d=\\"M11.302 12.831c.343-.34.537-.804.54-1.287V7.598h1.312v3.946a3.141 3.141 0 0 1-3.142 3.132h-5.46a3.141 3.141 0 0 1-3.14-3.132V6.075a3.132 3.132 0 0 1 3.14-3.132h3.79v1.312h-3.79a1.829 1.829 0 0 0-1.802 1.82v5.469a1.829 1.829 0 0 0 1.802 1.82h5.46c.484 0 .948-.192 1.29-.533Z\\"></path></svg>
             </div>
                 </article>"
 `;
 
-exports[`card.js should return the expected HTML 4`] = `
+exports[`card.js > should return the expected HTML 4`] = `
 "<article class=\\"c-card\\">
                     
                     
                     <div class=\\"c-card-labelContainer\\">
                 <a class=\\"c-card-label\\" href=\\"link.com\\" target=\\"_blank\\"><span>label</span></a>
-                <svg xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 16 16\\" height=\\"21\\" width=\\"21\\" fill=\\"#000\\" aria-hidden=\\"true\\" class=\\"c-pieIcon c-pieIcon--linkExternal\\"><path fill=\\"#242E30\\" d=\\"M13.25 3.564 8.21 8.63l-.928-.928 5.093-5.066H9.662V1.324h3.833a1.093 1.093 0 0 1 1.067 1.093V6.25H13.25V3.564Z\\"></path><path fill=\\"#242E30\\" d=\\"M11.302 12.831c.343-.34.537-.804.54-1.287V7.598h1.312v3.946a3.141 3.141 0 0 1-3.142 3.132h-5.46a3.141 3.141 0 0 1-3.14-3.132V6.075a3.132 3.132 0 0 1 3.14-3.132h3.79v1.312h-3.79a1.829 1.829 0 0 0-1.802 1.82v5.469a1.829 1.829 0 0 0 1.802 1.82h5.46c.484 0 .948-.192 1.29-.533Z\\"></path></svg>
+                <svg xmlns=\\"http://www.w3.org/2000/svg\\" role=\\"presentation\\" focusable=\\"false\\" fill=\\"#000\\" viewBox=\\"0 0 16 16\\" height=\\"21\\" width=\\"21\\" aria-hidden=\\"true\\" class=\\"c-pieIcon c-pieIcon--linkExternal\\"><path d=\\"M13.25 3.564 8.21 8.63l-.928-.928 5.093-5.066H9.662V1.324h3.833a1.093 1.093 0 0 1 1.067 1.093V6.25H13.25V3.564Z\\"></path><path d=\\"M11.302 12.831c.343-.34.537-.804.54-1.287V7.598h1.312v3.946a3.141 3.141 0 0 1-3.142 3.132h-5.46a3.141 3.141 0 0 1-3.14-3.132V6.075a3.132 3.132 0 0 1 3.14-3.132h3.79v1.312h-3.79a1.829 1.829 0 0 0-1.802 1.82v5.469a1.829 1.829 0 0 0 1.802 1.82h5.46c.484 0 .948-.192 1.29-.533Z\\"></path></svg>
```

Comment:
This appears to be adding some new attributes to the SVG is there a newer version of the icon being used or something?

Also just double checking that it seems like some of the path `fill` attributes have changed, is this expected?

Reply:
What was the resolution on this?